eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,291 +1,270 @@
-        str(n_multiple_evidence_strings) + ' ClinVar records generated more than one evidence_string',
-        str(n_germline_somatic) + ' ClinVar records with germline and somatic origins',
-        str(n_multiple_allele_origin) + ' ClinVar records with more than one allele origin',
-        'Number valid ClinVar records with unprocessed allele origins:'
-    ]
-
-    report_strings.extend([' ' + alleleOrigin + ': ' + str(n_unrecognised_allele_origin[alleleOrigin]) for alleleOrigin in n_unrecognised_allele_origin])
-
-    report_strings.extend([
-        str(no_variant_to_ensg_mapping) + ' ClinVar records with allowed clinical significance and valid rs id were skipped due to a lack of Variant->ENSG mapping.',
-        str(n_missed_strings_unmapped_traits) + ' ClinVar records with allowed clinical significance, valid rs id and Variant->ENSG mapping were skipped due to a lack of EFO mapping (see ' + config.UNMAPPED_TRAITS_FILE_NAME + ').',
-        str(n_records_no_recognised_allele_origin) + ' ClinVar records with allowed clinical significance, valid rs id, valid Variant->ENSG mapping and valid EFO mapping were skipped due to a lack of a valid alleleOrigin.',
-        str(n_more_than_one_efo_term) + ' evidence strings with more than one trait mapped to EFO terms',
-        str(len(unavailable_efo_dict)) + ' evidence strings were generated with traits without EFO correspondence',
-        str(n_valid_rs_and_nsv) + ' evidence strings were generated from ClinVar records with rs and nsv ids',
-        str(n_nsvs) + ' total nsvs found',
-        str(n_nsv_skipped_clin_sig) + ' ClinVar nsvs were skipped because of a different clinical significance',
-        str(n_nsv_skipped_wrong_ref_alt) + ' ClinVar nsvs were skipped because of same ref and alt'
-    ])
-
-    for line in report_strings:
-        print(line)
-
-
-def add_evidence_string(clinvarRecord, ev_string, evidence_string_list, n_evidence_strings_per_record):
-    try:
-        ev_string.validate()
-        evidence_string_list.append(ev_string)
-        n_evidence_strings_per_record += 1
-    except jsonschema.exceptions.ValidationError as err:
-        print('Error: evidence_string does not validate against schema.')
-        print('ClinVar accession: ' + clinvarRecord.accession)
-        print(err)
-        print(json.dumps(ev_string))
-        sys.exit(1)
-    except efo_term.EFOTerm.IsObsoleteException as err:
-        print('Error: obsolete EFO term.')
-        print('Term: ' + ev_string.get_disease().efoid)
-        print(err)
-        print(json.dumps(ev_string))
-        sys.exit(1)
-
-    return n_evidence_strings_per_record
+    return mappings
+
+
+def skip_record(clinvarRecord, cellbase_record, allowed_clinical_significance, counters):
+    if clinvarRecord.clinical_significance not in allowed_clinical_significance:
+        if clinvarRecord.nsv is not None:
+            counters["n_nsv_skipped_clin_sig"] += 1
+        print("Not in allowed")
```

Comment:
If these kind of debug statements are to be displayed they would need to show some context information: the record that failed, the incorrect values, etc.